package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "@octokit/core": "^4.0.0",
-    "@octokit/plugin-rest-endpoint-methods": "^7.0.0",
+    "@octokit/plugin-rest-endpoint-methods": "^8.0.0",
```

Comment:
actually the dependency is not used at all as far as I can tell? I wouldn't know how anyway. I think we can safely remove it altogether

```suggestion
```